mne/viz/_brain/_brain.py

```diff
@@ -1345,7 +1352,14 @@ def _on_button_release(self, vtk_picker, event):
         if self._mouse_no_mvt > 0:
             x, y = vtk_picker.GetEventPosition()
             # programmatically detect the picked renderer
-            self.picked_renderer = self.plotter.iren.FindPokedRenderer(x, y)
+            try:
+                # pyvista<0.30.0
+                self.picked_renderer = \
+                    self.plotter.iren.FindPokedRenderer(x, y)
+            except AttributeError:
+                # pyvista>=0.30.0
+                self.picked_renderer = \
+                    self.plotter.iren.interactor.FindPokedRenderer(x, y)
```

Comment:
Like this block works for >= 0.30 and < 0.30

Reply:
But this one does not depend on another package. 

If the user does not have `ipyvtklink` but has `pyvista 0.30.0`, it just won't work.

Reply:
That's why I think bump is the least painful solution